glustercli/cmd/volume-reset.go

```diff
@@ -0,0 +1,65 @@
+package cmd
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/gluster/glusterd2/pkg/api"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+const (
+	volumeResetCmdHelpShort = "Reset volume options"
+	volumeResetCmdHelpLong  = "Reset options on a specified gluster volume. Needs a volume name and at least one option"
+)
+
+var (
+	flagForce, flagResetAll bool
+
+	volumeResetCmd = &cobra.Command{
+		Use:   "reset <volname> <option>",
+		Short: volumeResetCmdHelpShort,
+		Long:  volumeResetCmdHelpLong,
+		Args:  cobra.RangeArgs(1, 2),
+		Run: func(cmd *cobra.Command, args []string) {
+			var req api.VolOptionResetReq
+			volname := args[0]
+			if flagForce {
+				req.Force = true
+			}
+			if flagResetAll {
+				req.All = true
+				req.Options = []string{}
+			} else {
+				if len(args) == 2 {
+					req.Options = append(req.Options, args[1])
+				} else if len(args) > 2 {
+					failure("Specify only one volume option at a time", errors.New("Specify only one volume option at a time"), 1)
```

Comment:
Well, is there a reason for this limitation - one at a time ? If not, we should support multiple at once.